src/app/components/ResultsList/ResultsList.jsx

```diff
@@ -109,86 +109,87 @@ const ResultsList = ({
     const resourceType = hasPhysicalItems ? 'item' : 'resource';
     const itemMessage = `${itemCount} ${resourceType}${itemCount !== 1 ? 's' : ''}`;
     return (
-      <li key={i} className={`nypl-results-item ${hasRequestTable ? 'has-request' : ''}`}>
-        <Heading level="three">
-          <Link
-            onClick={
+      <li>
+        <Card key={i} className={`nypl-results-item ${hasRequestTable ? 'has-request' : ''}`}>
```

Comment:
The `key` attribute should go in the `li` element like it was before. It's for React to keep track of the elements it renders in a list/array. The test also logs this warning:
<img width="895" alt="Screen Shot 2022-09-23 at 12 20 40 PM" src="https://user-images.githubusercontent.com/1280564/192007296-81b51233-b007-4afe-8af7-c8a0df1652fe.png">

Reply:
I'm not sure about the `className` because I don't know whether those styles need to go in the li or the `Card`.

Reply:
the classname was actually causing a bunch of tests to fail as well! thanks for pointing out.